### PR TITLE
Reduce communications load of making user connection

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
**Test Plan**

Check that you can make a new user connection and particle effects similar to before are displayed. (Spiralling sequence of dots is replaced by a non-spiralling disc of dots.)

Check that there's not a prolonged significant spike in Ethernet activity while both users continue to pull their triggers / press the "x" button to make a connection.


**Code Review**

Previously: Multitude entity edits were being made to the particle effects while a connection was being made (triggers pressed / "x" key held down): an edit each Script.update interval for as long as the connection was being made. A spike of ~1Mbps for a pair of avatars on a server, which would compound as there were more avatars present to distribute the edits to.

Now: A limited number of edits (circa 20) are made at a reduced rate (using Script.setTimeout). A near imperceptible spike in Ethernet traffic.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5588/HighFidelity-Beta-PR11967-380cd7aac6d322ca49171b02a173e019eea0321f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5588/HighFidelity-Beta-PR11967-380cd7aac6d322ca49171b02a173e019eea0321f.dmg

File: scripts/system/makeUserConnection.js [hunk 1]
```diff
@@ -111,10 +113,20 @@
...
+    var MINIMUM_PARTICLE_EMIT_RATE = 50;
+    var PARTICLE_DECAY_RATE = 0.5;
+    var particleEffectUpdateTimer = null;
+    var PARTICLE_EFFECT_UPDATE_INTERVAL = 200;
     var makingConnectionParticleEffect;
-    var makingConnectionEmitRate = 2000;
-    var particleEmitRate = 500;
+    var makingConnectionEmitRate;
+    var isMakingConnectionEmitting;
+    var INITIAL_MAKING_CONNECTION_EMIT_RATE = 500;
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply scripts/system/makeUserConnection.js]: Done

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5595/HighFidelity-Beta-PR11967-25214aad2368f2e9099fb5621fa1990abd2b281c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5595/HighFidelity-Beta-PR11967-25214aad2368f2e9099fb5621fa1990abd2b281c.exe

**Comment by Nex-Pro** [comment]
Testing Win10

**Comment by Nex-Pro** [comment]
Tested with 2X Win10 PC. Latest build of this PR.

### PASS

@ctrlaltdavid 

Tried this multiple times and checked network from both machines. Saw transitory spikes no greater than 0.1Mbs.

**Pull request merged**